torch/jit/__init__.py

```diff
@@ -423,58 +423,65 @@ def trace(*args, **kwargs):
         invocations of the model.
 
     Arg:
-        *args - a list of example tensors that will be passed to the function
-                as inputs while tracing. The resulting trace can be run with
-                inputs of different types and shapes assuming the traced operations
-                support those types and shapes.
+        func - a python function or torch.nn.Module that will be run with example_inputs.
```

Comment:
This generally doesn't follow our syntax for docstrings and probably won't render correctly in sphinx docs. The "Keyword arguments" section follows the right format.